src/UI/WebsiteAccountTab.java

```diff
@@ -43,6 +43,9 @@
 	private JButton websiteAccountSearchButton;
 	private DefaultListModel<String> websiteModel;
 
+	/**
+	 * @wbp.parser.entryPoint
```

Comment:
Is this so that you could pull up the editor window ?

Reply:
yeah should that be deleted?

Reply:
Maybe at the end but I was more just wondering bc I noticed the editor wouldn't come up since we split the file